cmake/globalconfig.cmake

```diff
@@ -207,17 +207,21 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 endif()
 
+# Runtime module loading
+option(IVW_CFG_RUNTIME_MODULE_LOADING 
+       "Load modules from dynamic libraries (dll/so) at application startup" OFF)
+
 if(UNIX AND NOT APPLE)
     set(CMAKE_POSITION_INDEPENDENT_CODE ON) # Will add -fPIC under linux.
     set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--as-needed") # Only link to libs as needed.
 endif()
 
-# Runtime module loading
-option(IVW_CFG_RUNTIME_MODULE_LOADING 
-       "Load modules from dynamic libraries (dll/so) at application startup" OFF)
+if(APPLE AND NOT OpenMP_CXX_INCLUDE_DIR)
+    set(OpenMP_CXX_INCLUDE_DIR /opt/homebrew/opt/libomp/include)
```

Comment:
Should there not be a check if the directory exists?
And then possibly instructions on how to install it via homebrew if missing